server/protocols/webadmin/webadmin-core/src/test/java/org/apache/james/webadmin/routes/TasksRoutesTest.java

```diff
@@ -111,6 +113,70 @@ void listShouldReturnTaskDetailsWhenTaskInProgress() throws Exception {
             .body("[0].cancelledFrom", nullValue());
     }
 
+    @Test
+    void listShouldBeSorted() throws Exception {
+        TaskId taskId1 = taskManager.submit(new MemoryReferenceTask(() -> Task.Result.COMPLETED));
+        Thread.sleep(10);
+        TaskId taskId2 = taskManager.submit(new MemoryReferenceTask(() -> Task.Result.COMPLETED));
+        Thread.sleep(10);
+        TaskId taskId3 = taskManager.submit(new MemoryReferenceTask(() -> Task.Result.COMPLETED));
+        Thread.sleep(10);
```

Comment:
why don't we single sleep after 3 submits?
```suggestion
        TaskId taskId1 = taskManager.submit(new MemoryReferenceTask(() -> Task.Result.COMPLETED));
        TaskId taskId2 = taskManager.submit(new MemoryReferenceTask(() -> Task.Result.COMPLETED));
        TaskId taskId3 = taskManager.submit(new MemoryReferenceTask(() -> Task.Result.COMPLETED));
        Thread.sleep(10);
```